src/passes/J2CLOpts.cpp

```diff
@@ -30,9 +30,56 @@ namespace wasm {
 
 namespace {
 
+using AssignmentCountMap = std::unordered_map<Name, Index>;
+using TrivialFunctionMap = std::unordered_map<Name, Expression*>;
+
 bool isOnceFunction(Function* f) { return f->name.hasSubstring("_<once>_"); }
 
-using AssignmentCountMap = std::unordered_map<Name, Index>;
+// Returns the function body if it is a trivial function, null otherwise.
+Expression* getTrivialFunctionBody(Module* m, Function* f) {
```

Comment:
```suggestion
Expression* getTrivialFunctionBody(Module* module, Function* func) {
```
(these are the conventions we have elsewhere)